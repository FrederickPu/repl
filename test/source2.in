{"cmd": "import Lean\ndef f := 1\ndef g := 2\n"}

{"src": 1}

{"src": 1, "after": true, "self": false, "before": false}

{"src": 1, "after": true, "before": false}

