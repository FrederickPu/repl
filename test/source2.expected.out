{"results":
 [{"source": "import Lean\n  \n  ", "env": 0},
  {"source": "def f :=\n  1", "env": 1},
  {"source": "def g :=\n  2", "env": 2}]}

{"src": "import Lean\n  \n  \n\ndef f :=\n  1"}

{"src": "def g :=\n  2"}

{"src": "def f :=\n  1\n\n\n\ndef g :=\n  2"}

